runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/FlinkStateInternals.java

```diff
@@ -75,7 +75,7 @@ public FlinkStateInternals(AbstractStateBackend flinkStateBackend, Coder<K> keyC
    * Returns the minimum over all watermark holds.
    */
   public Instant watermarkHold() {
-    long min = Long.MAX_VALUE;
+    long min = BoundedWindow.TIMESTAMP_MAX_VALUE.getMillis();
```

Comment:
BoundedWindow.TIMESTAMP_MAX_VALUE is the maximum value for any Beam timestamp. There is a ArithmeticException(The calculation caused an overflow) when use the Long.MAX_VALUE.